### DISCUSSION TITLE
food/projects/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/food#7] [General]
# food/projects/

https://shanenull.com/food/projects/

**Comment by shane0** [comment]
couscous black bean corn sounds yummy
https://www.masteringdiabetes.org/black-bean-corn-and-couscous-salad/#:~:text=Whole%20wheat%20couscous%20can%20also,white%20pasta%20or%20white%20rice.

**Comment by shane0** [comment]
https://www.fullcircle.com/

**Comment by shane0** [comment]
# documentary list

- What The Health (2017 Documentary)
-

**Comment by shane0** [comment]
Dosa

https://youtu.be/H5izGZ0qXMI?si=i5kyfjeS6NG_AoqC

**Comment by shane0** [comment]
# smokemeister

- nuts
- cheese
- egg
- seafood

**Comment by shane0** [comment]
# cast iron

## breakfast

- layer sliced banana and blueberries
- emulsify eggs in vitamix
- slice croisants soak in eggs
- add on top of banana
- add nuts 
- add oregon growers
- 400 degrees for 10 minutes
- remove lid
- smart oven set to toast 7 slices
